<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>758</width>
    <height>474</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>GUI Profiler</string>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>761</width>
     <height>501</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout" stretch="1,4">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="4,6">
      <item>
       <widget class="QListWidget" name="profiling_list"/>
      </item>
      <item>
       <widget class="PlotWidget" name="profiling_plot" native="true"/>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QTableView" name="profiling_table"/>
    </item>
   </layout>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>PlotWidget</class>
   <extends>QWidget</extends>
   <header>pyqtgraph</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
